src/mpi/coll/algorithms/common/algo_common.h

```diff
@@ -51,6 +51,15 @@ static inline int MPIR_Algo_calculate_pipeline_chunk_info(MPI_Aint chunk_size, M
     *num_segments = (count + *segsize_ceil - 1) / (*segsize_ceil);
 
   fn_exit:
+    MPL_DBG_MSG_FMT(MPIR_DBG_COLL, VERBOSE,
+                    (MPL_DBG_FDEST,
```

Comment:
We could possibly pass and preserve the calling function name as well. (e.g., "AllReduce", "Reduce" etc.). However, I'm not sure how important that is. Also it seems the function entry /exit log can add that info. Any thoughts?

Reply:
> ...it seems the function entry /exit log can add that info. Any thoughts?

This was my thinking for not including it. Just look at the previous log messages for context. Plus it doesn't add an argument argument to the pipeline calculation API that's unused by anything but the log output.

Reply:
That sounds fine.